torch/testing/_internal/common_device_type.py

```diff
@@ -1064,13 +1064,15 @@ def __init__(
         dtypes: Union[OpDTypes, Sequence[torch.dtype]] = OpDTypes.supported,
         allowed_dtypes: Optional[Sequence[torch.dtype]] = None,
         skip_if_dynamo=True,
+        sample_rules=None,
```

Comment:
```suggestion
        sample_specific_skips_and_xfails=None,
```
would be most precise lol

i do agree rules is vague. but i don't necessarily have better ideas 😛

Reply:
I think I'm good with `sample_skips_and_xfails` since that's clearer

Reply:
yea sgtm